src/lib/cloud-sdk-api.js

```diff
@@ -29,12 +29,22 @@ class CloudSdkAPI {
   /**
    * Initializes a CloudSdkAPI object and returns it.
    *
+   * @param {string} cmUrl the cloudmanager api endpoint
+   * @param {string} apiKey the cloudmanager api key
+   * @param {string} orgId the cloudmanager org id
+   * @param {string} devConsoleUrl the dev console url for the environment
    * @param {string} baseUrl the base URL to access the API
```

Comment:
With all the URLs, should we rename `baseUrl` to `rdeApiUrl` or something else that is clearer?

Reply:
makes sense